<def f='llvm/build/include/c++/v1/exception' l='275' ll='283' type='static void std::__throw_with_nested&lt;type-parameter-0-0, type-parameter-0-1, false&gt;::__do_throw(_Tp &amp;&amp; __t)'/>
<def f='llvm/libcxx/include/exception' l='275' ll='283' type='static void std::__throw_with_nested&lt;type-parameter-0-0, type-parameter-0-1, false&gt;::__do_throw(_Tp &amp;&amp; __t)'/>
